CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# How to contribute
+
+The NEST simulator is a scientific tool and as such it is never ready and constantly changing to meet the needs of novel neuroscientific endeavors. Here you find the most important information on how you can contribute to NEST. This document is an excerpt from our [developer space](https://nest.github.io/nest-simulator/), which provides more detailed information.
+
+## Getting started
+
+* Make sure you have a [GitHub account](https://github.com/signup/free)
+* The development workflow is based purely on pull requests. This [article](https://nest.github.io/nest-simulator/development_workflow) gives information on how to work with git and GitHub if you are new to it.
+
+## Making Changes
+
+* Create a topic branch from NEST master in your fork. Please avoid working directly on the `master` branch. See [issue 31](https://github.com/nest/nest-simulator/issues/31) for more information.
+* Make commits of logical units.
+* Make sure NEST compiles and has no new warnings.
+* Make sure all tests pass (`make installcheck`).
+* Make sure your code conforms to our [coding guidelines](https://nest.github.io/nest-simulator/coding_guidelines_c++#check-static-analysis-local) (`./extras/check_code_style.sh`)
+
+
+## Code Review
+
+We review each pull request according to our [code review guidelines](https://nest.github.io/nest-simulator/code_review_guidelines):
+
+* In general, the rule is that each pull request needs an OK from the CI platform and at least two reviewers to be merged.
```

Comment:
perhaps it's worth mentioning that *if* it is relevant to an issue then it should say so explicitly (now we have removed the line about requiring an issue number)